homeassistant/components/light/__init__.py

```diff
@@ -165,9 +178,15 @@ def handle_light_service(service):
         if transition is not None:
             params[ATTR_TRANSITION] = transition
 
+        service_fun = None
         if service.service == SERVICE_TURN_OFF:
```

Comment:
Yeah you're right, we shouldn't couple service names and attribute names.

Thought of 1 more optimization.
```python
if service.service == SERVICE_TURN_OFF:
    service_fun = light.turn_off
[…]

if service_fun:
    service_fun(**params)
```